tensorflow/lite/kernels/elementwise_test.cc

```diff
@@ -324,24 +400,103 @@ TEST(ElementWise, AbsInt16) {
               ElementsAreArray(ArrayFloatNear(abs_data, kQuantizedTolerance)));
 }
 
+TEST(ElementWise, AbsFloat16) {
+  ElementWiseOpFloatModel<Eigen::half> m(BuiltinOperator_ABS, {1, 1, 4, 1});
+  m.PopulateTensor<Eigen::half>(
+      m.input(), {Eigen::half(1.576170e+00), Eigen::half(5.097660e+00),
+                  Eigen::half(4.453130e+00), Eigen::half(4.390630e+00)});
+  ASSERT_EQ(m.Invoke(), kTfLiteOk);
+  EXPECT_THAT(m.ExtractVector<Eigen::half>(m.output()),
```

Comment:
ASSERT the output shape before checking the values

Reply:
Hi Luke, currently we are using `EXPECT_THAT` to check the output shape which is being used on all the test cases that are already present in the elementwise_test.cc file. Do I need to change it to `ASSERT_THAT` on all the newly added test cases?

Reply:
Addressed this review comment on [#e83c250](https://github.com/tensorflow/tensorflow/tree/e83c25039e6654a4a857d26a91627e83c0736aa5)